build.gradle

```diff
@@ -57,18 +57,19 @@ allprojects {
     }
   }
 
-  configurations.all {
-    resolutionStrategy {
-      force 'com.fasterxml.jackson:jackson-bom:2.13.4'
-      force 'com.fasterxml.jackson.core:jackson-databind:2.13.4'
-      force 'org.apache.orc:orc-core:1.8.3'
-      force 'com.google.guava:guava:31.1-jre'
+  if (it.path != ':integrations:spark-3.5:openhouse-spark-3.5-itest') {
+    configurations.all {
```

Comment:
can't this be overwritten in `openhouse-spark-3.5-itest`'s build.gradle? and leave the changes in this file be..

Reply:
The configuration takes precedence over the subproject's build file.